Source/bmalloc/bmalloc/TZoneHeapManager.cpp

```diff
@@ -79,51 +79,60 @@ void TZoneHeapManager::init()
 {
     RELEASE_BASSERT(m_state == TZoneHeapManager::Uninitialized);
 
-    if (verbose)
+    if constexpr (verbose)
         TZONE_LOG_DEBUG("TZoneHeapManager initialization ");
 
 #if BOS(DARWIN)
     // Use the boot UUID and the process' name to seed the key.
     static const size_t rawSeedLength = 128;
     char rawSeed[rawSeedLength] = { };
-    char bootDevUUID[NAME_MAX] = { };
-    size_t bootDevUUIDLen = sizeof(bootDevUUID);
 
-    // try using "kern.bootuuid" sysctl to obtain boot volume UUID
-    RELEASE_BASSERT(!sysctlbyname("kern.bootuuid", bootDevUUID, &bootDevUUIDLen, nullptr, 0));
```

Comment:
I see that we are blocking 'kern.bootuuid' in the sandboxes. How did this call succeed without that?

Reply:
I don't think it did.

Reply:
It worked on macOS but not on iOS.